extensions/bitwarden/src/authenticator.tsx

```diff
@@ -301,28 +302,40 @@ const authenticator = {
       if (error) return;
 
       const setTimeAndCode = () => {
-        const timeRemaining = Math.ceil(generator.remaining() / 1000);
-        setTime(timeRemaining);
-
-        if (timeRemaining === generator.period) {
-          setCode(generator.generate());
+        try {
+          const timeRemaining = Math.ceil(generator.remaining() / 1000);
+          setTime(timeRemaining);
+
+          if (timeRemaining === generator.period) {
+            setCode(generator.generate());
+          }
+        } catch {
+          const error = new Error("ERR2: Failed to regenerate");
+          setState(Err(error));
+          captureException(error.message, error, { captureToRaycast: true });
```

Comment:
This will send a simple `ERR2: Failed to regenerate` error to the `Extension Issues`.